### ISSUE TITLE
Training Data Issue

### ISSUE THREAD

**Issue by HamzaJavaid-gh** [HighwayWu/LASTED#1]
Hi!

I'm trying to train a custom dataset with your network with COCO and stable diffusion subset each 200K images. 
I tried with the r1 i.e [Real, Synthetic] with 0,1 labels in the text file ,first but the loss is not dropping. 

Then I tried to divide my data with r3 [Real, Synthetic, Real Painting, Synthetic Painting] each 200k images and assigned them labels of 0,1,2,3 accordingly. The loss increased instead with same BS. I see maybe there is some confusion in how to prepare the training file with labels. Can you provide a sample training txt file with image paths?

Thanks in advance

**Comment by HighwayWu** [comment]
Thanks for your asking. It is very strange that loss cannot decrease in both labeling (e.g., r1 or r3). Could you provide more detailed information? Also, try using pre-trained weights (e.g., ImageNet), which will accelerate convergence.

**Comment by HamzaJavaid-gh** [comment]
I've tried similar training data with r3, but if we can reproduce your results so if its possible to release the training data? 
That'd be very helpful

**Comment by HighwayWu** [comment]
Thanks for your attention. The training dataset will be released soon.

**Comment by HighwayWu** [comment]
The training dataset can be downloaded from the following link. Due to Google Drive's capacity limitations, we are currently only able to share it via Baidu Pan: https://pan.baidu.com/s/1ZgSiNX_Dd7cZcwiHv1ujVg?pwd=h4p7
